main.js

```diff
@@ -0,0 +1,7 @@
+console.log('frogs r radd');
+
+function printToDom(stuffToPrint, divId) {
```

Comment:
This will be good for all of us to have, but going forward, it would be better to save this until we have something to pass to it.